### PR TITLE
Move to Linux Foundation Charities 501(c)(3) and establish the Jupyter Foundation directed fund as part of Linux Foundation 501(c)(6)

### PR DESCRIPTION
**Pull request by Zsailer**
### Background, context and reasoning to help others understand the change.

The Jupyter Executive Council (EC) has the responsibility to ensure the healthy and sustainable evolution of Project Jupyter and to seek the best vehicles to support this. In early March, we, the EC, [proposed](https://jupyter.org/governance/linux-proposal.html) to create a new funding vehicle for the project, paired with some management and operational changes in support of this funding mechanism, in partnership with the Linux Foundation (LF). These changes will enable us to grow and develop ambitious efforts we have often discussed but lacked resources to implement, and provide long-term stability both for key Jupyter contributors and the project itself. While Project Jupyter has been impactful and successful in many areas, we currently struggle in our ability to scale in certain strategic directions and in maintaining core functions needed for an effort of our size and scope. We believe this plan is the right path forward to meet these challenges.

### A brief summary of the change.

The EC recommends that Project Jupyter transfers from NumFOCUS to the Linux Foundation Charities 501(c)(3). This transition would be paired with the creation of a new entity, the Jupyter Foundation, whose purpose is to raise, budget and spend funds in support of Project Jupyter and its mission. The Jupyter Foundation would be under the Linux Foundation 501(c)(6). 

If this proposal passes, as a project of [LF Charities](https://lf-charities.org/), Project Jupyter would adhere to the applicable LF Charities Policies as noted on the [LF Charities homepage](https://lf-charities.org).

A draft of the Jupyter Foundation participation agreement (which includes the Jupyter Foundation charter as Exhibit B) is included in this repo as part of this PR. For convenience, we also attach this document [here](https://github.com/user-attachments/files/16074730/jupyter_foundation_participation_agreement_draft-20240702.pdf). The EC and Linux Foundation may update this document as part of setting up the Jupyter Foundation, and a final version will be posted publicly.

### Alternatives to making this change and other considerations.

The EC explored other options that were described in the March proposal [here](https://jupyter.org/governance/linux-proposal.html#what-other-options-have-we-explored) and discussed further [here](https://github.com/jupyter/governance/issues/204). For more context and answers to various questions raised by the community, you can read the [FAQ here](https://jupyter.org/governance/linux-proposal.html#other-questions-faq).

## Voting

Vote is expected to close 15 July 2024 Anywhere on Earth.

### EC members/voting checkboxes

* @afshin
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
* @ellisonbg
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
* @jasongrout
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
* @fperez
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
* @Ruv7
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
* @zsailer
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain

### SSC members/voting checkboxes

- @gabalafou (Accessibility)
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
- @ivanov (Foundations and Standards)
  * [ ]  Yes
  * [x]  No
  * [ ]  Abstain
- @JohanMabille (Kernels)
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
- @jtpio (Frontends)
  * [ ]  Yes
  * [ ]  No
  * [x]  Abstain
- @marthacryan (DEI)
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
- @martinRenou (Voilà)
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
- @minrk (JupyterHub and Binder)
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
- @rpwagner (Security)
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain
- @SylvainCorlay (Widgets)
  * [ ]  Yes
  * [x]  No
  * [ ]  Abstain
- @vidartf (Server)
  * [x]  Yes
  * [ ]  No
  * [ ]  Abstain

> ## The process ❗
> 
> The process for changing the governance pages is as follows:
> 
> * ✅ Open a pull request **in draft state**. This triggers a discussion and iteration phase
>   for your proposed changes.
> * ✅  When you believe enough discussion has happened,
>   **move the pull request to an active state**. This triggers a vote.
> * During the voting phase, no substantive changes may be made to the pull request.
> * The Executive Council and Software Steering Council will vote, and at the end of voting the pull request is merged or closed.
> 
> The discussion phase is meant to gather input and multiple perspectives from the community.
> Make sure that the community has had an opportunity to weigh in on
> the change **before calling a vote**. A good rule of thumb is to ask several Council
> members if they believe that it is time for a vote, and to let at least one person review
> the pull request for structural quality and typos.

### CODE REVIEW

File: jupyter_foundation.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Jupyter Foundation
+
+The Jupyter Foundation is a directed fund of the Linux Foundation 501(c)(6). It exists to provide resources and strategic counsel to Project Jupyter. The charter is here (TODO: Link). We summarize relevant points from the charter below.
```
Comment (jasongrout) [inline jupyter_foundation.md:3]: FYI, we still need to link to a draft of the charter (which we are close to having a draft of, and will be available for review before we vote). Also, we should clarify here that the charter is the source of truth for this information, and any discrepancy between this document and the charter should defer to the charter.

File: jupyter_foundation.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- Develop and approve an annual strategic plan and budget for funding from the Jupyter Foundation to benefit Project Jupyter.
+- Recruit and retain Jupyter Foundation members.
+
+## Membership
+
+The Jupyter Foundation Governing Board consists of:
+
+- All members of the [Jupyter Executive Council](./executive_council.md)
+- One representative from each Jupyter Foundation Premier Member
+- No more than two representatives total from all Jupyter Foundation General Members
```
Comment (jasongrout) [inline jupyter_foundation.md:23]: The current draft of the charter has:
```suggestion
- Between one and three representatives total from all Jupyter Foundation General Members
```

File: people.md [hunk 3]
```diff
@@ -27,6 +27,18 @@ Alphabetical by first name, names are followed by GitHub usernames.
 | JupyterHub and Binder | Min Ragan-Kelley | [@minrk](https://github.com/minrk) |
 | Voilà | Martin Renou | [@martinRenou](https://github.com/martinRenou) |
 
+## [Jupyter Foundation](jupyter_foundation.md) Governing Board
```
Comment (jasongrout) [inline people.md:30]: To be filled out with other members over time. But we can start with the EC right now.

**Comment by jasongrout** [comment]
We combined several refactoring/reordering cleanups with this proposal (my fault...). Should we remove those so the actual proposal really stands out on its own?

These cleanups were done:

- Moving the old governance file to an `archive/` directory
- Reordering the subproject and distinguished contributor sections of the overview

File: overview.md [hunk 4]
```diff
@@ -1,37 +1,38 @@
...
+3. The [**Jupyter Foundation**](./jupyter_foundation.md) is a directed fund of the Linux Foundation 501(c)(6) that exists to provide resources and strategic counsel to Project Jupyter. The Executive Council serves on the Jupyter Foundation Governing Board. For more details, see the [Jupyter Foundation document](./jupyter_foundation.md).
 
 Additionally, the Executive Council (EC) receives input from a Community Advisory Panel. This panel advises the EC with perspectives and connections that may reach beyond the active Jupyter community.
 
 ## Other major components of the organization
 
 In addition to these three bodies, the following are other major parts of the Project related to governance.
 
-### Distinguished Contributors
+### Software Subprojects
```
[the remark this answers is not in the document]
Comment (Zsailer) [reply overview.md:21]: I've opened #228 to handle this re-ordering. I will update this PR so that we don't do any re-ordering here and can keep this PR focused on the LF updates.

**Comment by Zsailer** [comment]
@jasongrout I opened #228 to address the cleanup you referenced. Let's decouple those changes from this PR.

File: jupyter_foundation.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Decision making
+
+The Jupyter Foundation Governing Board follows the decision-making policies in its charter. (TODO: Link)
```
Comment (ellisonbg) [inline jupyter_foundation.md:27]: To resolve any ambiguities here between our decision making policies and those of the LF, in addition to providing a link, we should clarify that this is the "Jupyter Decision Making policy".

File: jupyter_foundation.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# Jupyter Foundation
+
+The Jupyter Foundation is a directed fund of the Linux Foundation 501(c)(6). It exists to provide resources and strategic counsel to Project Jupyter. The charter is here (TODO: Link). We summarize relevant points from the charter below.
+
+
+## Jupyter Foundation Governing Board
```
Comment (fperez) [inline jupyter_foundation.md:6]: I unfortunately missed the last EC meetings due to travel/illness, so forgive me if this was discussed to death already. But my inclination would be to name this "something something Council" instead of "Governing Board" to keep in line with our standard naming practices for most of our decision making bodies of this type.

If there's a good reason to divert from our practice, we should encode that in the [FAQ](https://jupyter.org/governance/linux-proposal.html) (right now I see no mention of the word "Board" there).

File: jupyter_foundation.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+The Jupyter Foundation Governing Board controls the resources of the Jupyter Foundation and works to build a healthy collaboration between the Jupyter community, Jupyter leadership, and members of the Jupyter Foundation, including for-profit organizations. The current members of the Jupyter Foundation Governing Board are listed in the [Leadership Directory](people.md#jupyter-foundation-governing-board).
+
+## Responsibilities of the Jupyter Foundation Governing Board
+
+- Serve Project Jupyter and its mission.
+- Meet regularly to review strategic objectives of Jupyter.
+- Develop and approve an annual strategic plan and budget for funding from the Jupyter Foundation to benefit Project Jupyter.
+- Recruit and retain Jupyter Foundation members.
```
Comment (fperez) [inline jupyter_foundation.md:15]: Not sure about the ideal wording, but I think we should explicitly list budgetary decisions/allocations on other potential timescales. I don't imagine that a single annual plan/budget would be the only mechanism for allocating funds we should have.

Perhaps something like 

> "Meet at least quarterly and allocate budgetary resources from the fund as appropriate, based on both the annual strategic plan and potential needs and opportunities that may arise and be brought up to the Board for consideration". 

Probably too wordy, but it's a start...

File: jupyter_foundation.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+## Membership
+
+The Jupyter Foundation Governing Board consists of:
+
+- All members of the [Jupyter Executive Council](./executive_council.md)
```
Comment (fperez) [inline jupyter_foundation.md:21]: Missing periods at the end of these items.

File: people.md [hunk 9]
```diff
@@ -27,6 +27,18 @@ Alphabetical by first name, names are followed by GitHub usernames.
...
+
+| Representative | Organization | GitHub username |
+| -------------- | ------------ | --------------- |
+| Afshin Darian | Executive Council | [@afshin](https://github.com/afshin) |
+| Ana Ruvalcaba | Executive Council | [@Ruv7](https://github.com/Ruv7) |
+| Brian Granger | Executive Council | [@ellisonbg](https://github.com/ellisonbg) |
+| Fernando Perez | Executive Council | [@fperez](https://github.com/fperez) |
```
Comment (fperez) [inline people.md:37]: Pérez (w/accent)

**Review by fperez — APPROVED** [review]

**Comment by fperez** [comment]
Very minor feedback from me, I think only one slightly substantial suggestion, but I don't think it's controversial...

Thanks all for the hard work here!

File: jupyter_foundation.md (hunk 8 above)
Comment (Zsailer) [reply jupyter_foundation.md]: Fixed.

File: people.md (hunk 9 above)
Comment (Zsailer) [reply people.md]: Fixed. I also updated it in other places across this page.

File: people.md (hunk 3 above)
Comment (Zsailer) [reply people.md:30]: :+1:

File: jupyter_foundation.md (hunk 6 above)
Comment (fperez) [reply jupyter_foundation.md:6]: Just discussed it, keeping it as "Jupyter Foundation Governing Board" b/c:

1. That's the standard terminology already used by LF in many other places, and it will benefit us greatly when fundraising for this specific body to use established patterns.
2. This is really a new type of group, whose job is to govern our new fund under the 501(c)6. We've never had anything quite like that, so it is actually useful to signal its difference via naming.

This makes sense to me, resolving.

File: jupyter_foundation.md (hunk 5 above)
Comment (fperez) [reply jupyter_foundation.md:27]: Other than adding the links (coming soon) we think this is clear enough - its charter spells out its specific decision making process, which is fairly straightforward but has more formal voting requirements in order to keep a clear record. I think it's unambiguous and we don't need to repeat it here, people can read the doc.

**Comment by jasongrout** [comment]
FYI, I added a short paragraph to the original description above that links to the LF Charities polices we would follow. These policies outline the procedure for leaving LF Charities, the G&A fee for funds raised, etc.

File: jupyter_foundation.md (hunk 5 above)
Comment (jasongrout) [reply jupyter_foundation.md]: I've clarified in a new commit here that the Jupyter Foundation Governing Board follows the decision-making guidelines in the Jupyter Foundation charter.

File: jupyter_foundation.md (hunk 1 above)
Comment (jasongrout) [reply jupyter_foundation.md]: Ana and I pushed a few changes, including adding a sentence around this.

File: jupyter_foundation.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+The Jupyter Foundation is a directed fund of the Linux Foundation 501(c)(6) whose purpose is to raise, budget and spend funds in support of Project Jupyter and its mission. The official Jupyter Foundation charter is here (TODO: link). A brief summary of relevant points from the charter is provided below.
+
+
+## Jupyter Foundation Governing Board
+
+The Jupyter Foundation Governing Board controls the resources of the Jupyter Foundation and works to build a healthy collaboration between the Jupyter community, Jupyter leadership, and members of the Jupyter Foundation, including for-profit organizations. The current members of the Jupyter Foundation Governing Board are listed in the [Leadership Directory](people.md#jupyter-foundation-governing-board).
+
+## Responsibilities of the Jupyter Foundation Governing Board
+
+- Serve Project Jupyter and its mission.
```
Comment (jasongrout) [inline jupyter_foundation.md:12]: To copy language from the draft Jupyter Foundation charter, let's change "Serve" to "Support"
```suggestion
- Support Project Jupyter and its mission.
```

**Comment by jasongrout** [comment]
FYI, here is a work-in-progress draft of the Jupyter Foundation participation agreement (i.e., what a corporation or other entity will sign to become a member of the Jupyter Foundation). Exhibit B in this document is the work-in-progress draft of the Jupyter Foundation charter, and exhibit C contains work-in-progress draft amounts for each membership tier. The EC and LF are continuing to iterate and refine these documents.
[Jupyter Foundation Participation Agreement and Funding Charter Draft 6-18-2024-1.pdf](https://github.com/user-attachments/files/15991398/Jupyter.Foundation.Participation.Agreement.and.Funding.Charter.Draft.6-18-2024-1.pdf)

File: jupyter_foundation.md (hunk 10 above)
Comment (Zsailer) [reply jupyter_foundation.md:12]: :+1:

**Comment by jasongrout** [comment]
> FYI, here is a work-in-progress draft of the Jupyter Foundation participation agreement (i.e., what a corporation or other entity will sign to become a member of the Jupyter Foundation). Exhibit B in this document is the work-in-progress draft of the Jupyter Foundation charter, and exhibit C contains work-in-progress draft amounts for each membership tier. The EC and LF are continuing to iterate and refine these documents.

A draft of the Jupyter Foundation participation agreement (which includes the Jupyter Foundation charter) is now included in the PR and linked to from the governance Jupyter Foundation document.

**Comment by jasongrout** [comment]
@jupyter/software-steering-council @jupyter/executive-council - we have opened the vote on this proposal. We expect to close voting on 15 July 2024, Anywhere on Earth.

**Comment by gabalafou** [comment]
Some context on my vote.

I reached out to the Jupyter Accessibility council about this issue in various ways starting back in April. I did not hear any objections from anybody on the Jupyter Accessibility council about the Linux Foundation proposal. As such, I am voting yes on the proposal. My reasoning is based on everything I've heard in various meetings and calls. I have been told that the governance and self-determination of Jupyter will be maintained. I am not convinced that the move will improve things on the accessibility front, but I haven't heard anything that makes me believe it would make things worse. 

Congratulations to everyone who has worked hard for this.

**Comment by fperez** [comment]
Hi all,

In response to some questions that have come up, let us (the EC) try to provide a summary of how we've navigated this process. This vote is _not_ a referendum on NumFOCUS (NF), but about the future of Jupyter. Ultimately what we are doing is _identifying how to best serve Jupyter_ at a time of significant resource challenges but also great opportunity for the project to grow, given the right partners.

-   Jupyter needs to substantially grow its funding base, including industry participation. We explored multiple options across the entire landscape of open source fiscal support, and we found the entire package of the Linux Foundation's (LF) infrastructure to be an excellent match to our needs (and no other fiscal sponsor we looked at to come close to being a fit).

-   Once that was clear, we proposed to NF that the entire ecosystem join together under LF as a single "Open source & open science" community, including Jupyter. We had multiple discussions over months with both LF and NF teams about this idea. Ultimately NF came forward with their own proposal to create a new 501(c)6. The NF 501(c)(6) is only in the beginning stages of discussion, and will likely take a while to be functional.

-   We, the Jupyter EC, have as our main responsibility to serve Jupyter. We have very limited bandwidth and are already leaving on the floor important Jupyter-specific needs, so we had to make a choice about how to use our energy.

-   We need resources _today_, as there are many unmet needs in the project. We think it is our responsibility to find a path towards that goal as soon as possible. We recognize we're part of a broader ecosystem, and we tried our best to support those discussions. But at the end of the day, we have no authority over NF or any other projects, and we need to meet our own duties first. It's one of  those "put your mask on before helping others in the plane" situations, and we are trying to do right by Jupyter first and foremost.

-   Finally, NF has acknowledged difficulties in operational matters that they intend to address in the future. For a project our size, including ambitious events programs and hiring staff, those difficulties have been more pronounced. We simply have to make a choice on where to go _now_, and while it's possible that in a few years NF will have addressed all these issues, built a 501(c)6 and successfully engaged with industry partners to fund it, Jupyter can't wait that long.

We hope this helps, especially the subprojects and their SSC representatives. As a reminder, we have [public EC office hours on Thursday at 10am Pacific on Zoom](https://zoom.us/j/2264645576?pwd=c0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09), and we'd be happy to talk then!

Fernando, on behalf of the EC (jointly discussed on July 9th).

**Comment by vidartf** [comment]
Just a reminder that today (AoE) is the last day of voting here.

**Comment by Zsailer** [comment]
This vote officially **passes**. :rocket: Thank you, all, for voting!

The final vote count:
* Executive Council: 6 yes.
* Software Steering Council: 7 yes, 2 No, 1 Abstain.

**Comment by fperez** [comment]
I wanted to drop a quick word to thank everyone! We had a lot of discussion on this topic and we really appreciate the thorough vetting the community gave to this question. It's an important step for our project, that I am convinced will help us grow and meet new challenges. Lots of work ahead, but I think it's an exciting moment!
